website/docs/faqs/Accounts/transfer-account.md

```diff
@@ -0,0 +1,24 @@
+---
+title: How do I transfer account ownership to another user? 
+description: "Instructions on how to transfer your dbt Cloud user account to another user"
+sidebar_label: 'How to transfer dbt Cloud account?'
+id: transfer-account
+
+---
+
+You can transfer your dbt Cloud [access control](/docs/collaborate/manage-access/about-access) to another user by following the steps below, depending on your dbt Cloud account plan:
+
+| Account plan| Steps |
+| ------ | ----------- |
+| **Developer** |  You can transfer ownership by changing the email directly on your dt Cloud [profile page](https://cloud.getdbt.com/#/profile/).|
+| **Team** | Existing account admins with account access can add or remove users to the owner group.
+| **Enterprise** | Account admins can add or remove users to a group with Account Admin permissions. |
+| **If all account owners left the company** | If the account owner has left your organization, you will need to work with _your_ IT department to have incoming emails forwarded to a new person. Once you have redirected the email, you can request to reset the user password and once logged in - you can change the email on the [Profile page](https://cloud.getdbt.com/#/profile/). | 
+
+**Still need help?** <br />
+If you're still experiencing issues, we'll be happy to help. Email our Support team at support@getdbt.com and provide us with the below info:
```

Comment:
I have a few concerns about this part. If a security professional were to read this FAQ I think this might throw up some red flags.  Does the email need to come from the same domain, a person of a particular position, or is there something they need to provide to validate their request to transfer an admin?  If there are any other checks that need to be included I think we should elaborate here so that it doesn't appear that anyone can just send an email and get admin access to an account with sensitive data switched.

Reply:
I agree.  The "Still Need Help" section should be omitted entirely.  There is no other/better answer.  Get your IT department to redirect the emails, full stop.  It's not hard and anyone can do it.  If they choose not to...separate problem.  😉